arangod/Indexes/IndexFactory.cpp

```diff
@@ -380,9 +380,6 @@ Result IndexFactory::validateFieldsDefinition(VPackSlice definition,
   auto const fieldMaybeObject = Index::TRI_IDX_TYPE_INVERTED_INDEX == idxType;
 
   if (fieldsSlice.isArray()) {
-    std::regex const idRegex("^(.+\\.)?" + StaticStrings::IdString + "$",
```

Comment:
That will disable check for all indexes. but we need only for inverted

Reply:
Done